### ISSUE TITLE
[Fix] title overlaps in accuracy figure of parameter estimation

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#575]
## Summary
Figure title overlaps in the output of `Scenario.estimate_accuracy(phase)`. This is related to #570.

## Codes
```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
jhu_data = data_loader.jhu()
population_data = data_loader.population()
# Scenario analysis
National_scenario = cs.Scenario(jhu_data, population_data, country="Italy", province= None, tau=1440)
# Fix the first date of records
National_scenario.first_date = "04May2020"
# Fix the last date of records
National_scenario.last_date = "20Oct2020"
National_scenario.trend(show_figure=False)
National_scenario.estimate(cs.SIR, phases=["6th"])
National_scenario.estimate_accuracy(phase="6th")
```

## Outputs
![Figure_1](https://user-images.githubusercontent.com/7270139/106353379-74d92c80-632d-11eb-9b4f-d86e7321dc20.png)

## Environment
- CovsirPhy version: 2.15.0-zeta
- Python version: 3.8.5
- Installation: poetly
- System: WSL (Ubuntu)

**Comment by lisphilar** [comment]
Title will be changed from "Comparison of observed/estimated {v}(t)" to "Comparison regarding {v}(t)".

**Comment by lisphilar** [comment]
Closed with #576.
